apps/api/src/app/execution-details/execution-details.controller.ts

```diff
@@ -36,4 +46,27 @@ export class ExecutionDetailsController {
       })
     );
   }
+
+  @Get('/transaction/:transactionId')
```

Comment:
I am not sure about the structure of this endpoint. The beginning with  '/transaction' I mean. I wonder if we should maybe do transactionId as a parameter  or maybe only '/:transactionId' 
@djabarovgeorge do you have a suggestion?

Reply:
@ainouzgali Great suggestion. Looking at it, that makes more sense.

Looking at the base get endpoint, the main difference between this new one and that one is that we're filtering by `transactionId` so `/:transactionId` seems like a good structure and we've already implemented it that way in the latest commit.

Reply:
The path should be pluralised as per REST design.
```/transactions/:transactionId```.